app/scripts/components/exploration/dataset-options.tsx

```diff
@@ -48,16 +52,22 @@ export default function DatasetOptions(props: DatasetOptionsProps) {
       </DropMenu>
       <DropMenu>
         <li>
-          <DropMenuItemButton
-            variation='danger'
-            onClick={() => {
-              setDatasets((datasets) =>
-                datasets.filter((d) => d.data.id !== dataset.data.id)
-              );
-            }}
+          <Tip
+            disabled={datasets.length > 1}
+            content="It's not possible to remove the last dataset. Add another before removing this one."
           >
-            <CollecticonTrashBin /> Remove dataset
-          </DropMenuItemButton>
+            <RemoveButton
```

Comment:
Found it a little bit odd that the disabled "Remove dataset" button get's a highlight effect on hover (I guess it's related to the tooltip).

![Screenshot 2023-09-04 at 10 30 05](https://github.com/NASA-IMPACT/veda-ui/assets/1583415/4430d784-7453-4a8a-b875-d6e0f22b5d54)

Reply:
Good catch! Fixed! :)